CMakePresets.json

```diff
@@ -4,17 +4,25 @@
     {
       "name": "base",
       "displayName": "Base Configuration",
-      "description": "Base build using Ninja generator and vcpkg",
+      "description": "Base build using Ninja generator",
       "generator": "Ninja",
       "binaryDir": "${sourceDir}/build/${presetName}",
+      "hidden": true
+    },
+    {
+      "name": "base_vcpkg",
```

Comment:
Can you rename `base` to `base-vcpkg` and introduce `base-syslibs`? I don't like deep inheritance, and a few copy pasted lines are much better than coupling.

So existing presets would extend from `base-vcpkg` and your new presets from `base-syslib`.

Also, vcpkg is the main build method, the syslib way is an alternative, so I don't really like it that `base_vcpkg` derives from `base`. Thanks 😄